### PR TITLE
docs: Added Ruby and Python for audio capture, add mobile command

### PR DESCRIPTION
**Pull request by KazuCocoa**
## Proposed changes

- Added Ruby and Python code for  audio capture
- Added audio command in mobile commands and refer to the documentation

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/en/writing-running-appium/ios/audio-capture.md [hunk 1]
```diff
@@ -42,6 +42,24 @@ try (FileOutputStream fos = new FileOutputStream("out.mp4")) {
...
 ```
 
+```ruby
+# Ruby
+@driver.execute_script 'mobile: startAudioRecording', audioInput: ':0'
+sleep 10000
+base64_str = @driver.execute_script 'mobile: stopAudioRecording'
+File.write 'out.mp4', Base64.decode64(base64_str)
+```
+driver.execute_script("mobile: scroll", {"direction": "down", "element": element.id})
```
Comment (ki4070ma) [inline docs/en/writing-running-appium/ios/audio-capture.md:52]: Unnecessary?

File: docs/en/writing-running-appium/ios/audio-capture.md [hunk 2]
```diff
@@ -42,6 +42,24 @@ try (FileOutputStream fos = new FileOutputStream("out.mp4")) {
 }
 ```
 
+```ruby
+# Ruby
+@driver.execute_script 'mobile: startAudioRecording', audioInput: ':0'
+sleep 10000
```
Comment (ki4070ma) [inline docs/en/writing-running-appium/ios/audio-capture.md:48]: Should be `10` ?

```
Thread.sleep(10000);  // milliseconds@java
```

File: docs/en/writing-running-appium/ios/audio-capture.md [hunk 3]
```diff
@@ -42,6 +42,24 @@ try (FileOutputStream fos = new FileOutputStream("out.mp4")) {
...
+base64_str = @driver.execute_script 'mobile: stopAudioRecording'
+File.write 'out.mp4', Base64.decode64(base64_str)
+```
+driver.execute_script("mobile: scroll", {"direction": "down", "element": element.id})
+
+
+```python
+# Python
+driver.execute_script('mobile: startAudioRecording', {'audioInput': ':1'})
+time.sleep(10000)
```
Comment (ki4070ma) [inline docs/en/writing-running-appium/ios/audio-capture.md:58]: Should be `10`?

File: docs/en/writing-running-appium/ios/audio-capture.md [hunk 4]
```diff
@@ -42,6 +42,24 @@ try (FileOutputStream fos = new FileOutputStream("out.mp4")) {
...
+base64_str = driver.execute_script('mobile: stopAudioRecording')
```
Comment (mykola-mokhnach) [inline docs/en/writing-running-appium/ios/audio-capture.md:59]: is it a string or byte array? Python3 won't allow to use 'wb' mode with a string without decoding it first

**Review by mykola-mokhnach — APPROVED** [review]

File: docs/en/writing-running-appium/ios/audio-capture.md (hunk 1 above)
Comment (KazuCocoa) [reply docs/en/writing-running-appium/ios/audio-capture.md]: yeah, my wrong :P

**Pull request merged**